src/features/subscription/helpers/useThematicSubscription.tsx

```diff
@@ -53,7 +55,10 @@ export const useThematicSubscription = ({
     async () => {
       analytics.logNotificationToggle(!!state.allowEmails, !!state.allowPush)
       if (!isSubscribeButtonActive) {
+        analytics.logSubscriptionUpdate({ from: 'thematicHome', type: 'in', entryId: homeId })
         await onUpdateSubscriptionSuccess?.()
+      } else {
+        analytics.logSubscriptionUpdate({ from: 'thematicHome', type: 'out', entryId: homeId })
```

Comment:
Perso j'aurai regrouper les log au même endroit, mais c'est juste une préférence : 
```suggestion
      analytics.logNotificationToggle(!!state.allowEmails, !!state.allowPush)
      analytics.logSubscriptionUpdate({
        from: 'thematicHome',
        type: isSubscribeButtonActive ? 'out' : 'in',
        entryId: homeId,
      })
      if (!isSubscribeButtonActive) {
        await onUpdateSubscriptionSuccess?.()
      }
```

Reply:
Je trouve ça bien comme on a déjà le if qui fait la diff de le réutiliser ahah